final_task/rss_reader/functions/process_func.py

```diff
@@ -0,0 +1,117 @@
+"""Contain parsing and text processing functions
+
+Functions
+---------
+parse_feed(command_line_args, logger) -> feed
+    Get feed by url using feedparser
+--------------------------------------------
+get_xml_root(xml_file) -> root
+    Create lxml parser, return root of xml file.
+    Function uses inside process_feed() function
+--------------------------------------------
+extract_text_from_description(root)-> text
+    Extract text from description root
+    Function uses inside process_feed() function
+--------------------------------------------
+extract_links_from_description(root):
+    Extract links from description
+    Function uses inside process_feed() function
+--------------------------------------------
+process_feed(feed, logger) -> news_collection
+    Get feed title
+    Create a list of news instance objects
+"""
+
+
+import feedparser
+from lxml import etree
+from classes.news_class import News
+import functions.check_func as ch_f
+
```

Comment:
Here and below: It is better to use `from <module> import <class/function>` in such situations.

Reply:
Fixed